R/telemetry.R

```diff
@@ -112,7 +112,9 @@ Telemetry <- R6::R6Class( # nolint object_name_linter
     #' @param track_anonymous_user flag that indicates to track anonymous user.
     #' A cookie is used to track same user without login over multiple sessions,
     #' This is only activated if none of the automatic methods produce a username
-    #' and when a username is not explicitly defined.`TRUE` by default
+    #' and when a username is not explicitly defined.`TRUE` by default.
+    #' @param track_errors flag that indicates if the basic telemetry should
+    #' track the errors. `TRUE` by default
```

Comment:
Please consider adding here a description that there is a limitation on the errors' information on versions before `shiny 1.8.1`.